api4/user.go

```diff
@@ -558,12 +558,14 @@ func getFilteredUsersStats(c *Context, w http.ResponseWriter, r *http.Request) {
 	channelID := r.URL.Query().Get("in_channel")
 	includeDeleted := r.URL.Query().Get("include_deleted")
 	includeBotAccounts := r.URL.Query().Get("include_bots")
+	includeRemoteUsers := r.URL.Query().Get("include_remote_users")
```

Comment:
Can you also link the webapp PR for this?

Reply:
I going to add the frontend PR for that soon, not 100% ready yet